source/howto/tourism/alpinebits.rst

```diff
@@ -0,0 +1,121 @@
+ 
+.. _ab-matrix:
+
+.. table:: Matrix of the protocol versions and supported actions in
+   AlpineBits.
+
+   +---------------------+---------+---------+---------+---------+
+   | AlpineBits Server   | 2017-10 | 2017-10 | 2018-10 | 2018-10 |
+   | Actions             |   PUSH  |   PULL  |   PULL  |   PUSH  |
+   +---------------------+---------+---------+---------+---------+
+   | FreeRooms           | Yes     | No      | Yes     | No      |
+   +---------------------+---------+---------+---------+---------+
+   | GuestRequests       | No      | No      | No      | No      |
+   +---------------------+---------+---------+---------+---------+
+   | Inventory Basic     | Yes     | Yes     | Yes     | Yes     |
+   +---------------------+---------+---------+---------+---------+
+   | Inventory HotelInfo | Yes     | Yes     | Yes     | Yes     |
+   +---------------------+---------+---------+---------+---------+
+   | RatePlans           | No      | No      | No      | No      |
+   +---------------------+---------+---------+---------+---------+
+   | BaseRates           | No      | No      | No      | No      |
+   +---------------------+---------+---------+---------+---------+
+
+Note that PUSH and PULL refer to the action of uploading to and
+downloading from AlpineBits servers, respectively.
+   
+Finally, to retrieve data from the AlpineBits, you need to set the
+correct content type and provide an `action`. The content type is
+specified in another header by :literal:`--header 'Content-Type:
+multipart/form-data'`, while the action is given as a form:
+:literal:`--form 'action=getVersion'`.
+
+.. _ab-request:
+
+Summing up what was described above, a call to the AlpineBits endpoint
+looks like the following one:
+	 
+.. code-block:: 
+		
+   curl --location --request POST \
+   'https://alpinebits.opendatahub.bz.it/AlpineBits' \
+   --header 'Authorization: Basic <your-token-here>' \
+   --header 'X-AlpineBits-ClientProtocolVersion: 2017-10' \
+   --header 'X-AlpineBits-ClientID: 'My test request' \
+   --header 'Content-Type: multipart/form-data' \
+   --form 'action=getVersion'
+
+Here, you can see that the additional option :literal:`--location` is
+used, that will make sure to resend the request in case a
+:strong:`3xx` HTTP error code is received, i.e., if the requested
+resource has been moved.
+
+Postman Setup
+-------------
+
+In Postman, you need to enter the same data as in the call shown in
+previous section. The Postman setup equivalent to that call is shown
+in the two screenshots.
+
+.. _ab-postman-header:
+
+.. figure:: /images/postman/AB-postman1.png
+
+   Definition of the call's headers.
+
+In the headers you need to add all the parameters as in the curl
+version, except for the authentication: this option need to be
+specified in the `Authorization` tab of postman. Here, choose
+:strong:`Basic Auth` as type and use :strong:`someuser` and
+:strong:`secret` as username and password, respectively.
```

Comment:
We do suggest to use some contact details here as well, in case we need to contact you for technical reasons.

Reply:
I added a note for this.